bot_scripts/delete_references.py

```
@@ -0,0 +1,78 @@
+    """
+    results = execute_sparql_query(query)
+    target_prop_dict = defaultdict(set)
+    for binding in results["results"]["bindings"]:
+        item = binding["item"]["value"]
+        item = item.split("/")[-1]
+        prop = binding["property"]["value"].split("/")[-1]
+        target_prop_dict[item].add(prop)
+    for k, v in target_prop_dict.items():
+        qids_to_remove = {qid}
+
+        for attempt in range(10):  # safety cap; usually 1-2 iterations
```

Ruff (0.15.12)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)